Added Typescipt, FastAPI and Django to the list of languages

prathamhanda opened the pull request:
### 🛠️ Pull Request Summary: Added Support for TypeScript, FastAPI & Django

This update introduces `.gitignore` support for three widely used technologies:

- TypeScript
- FastAPI
- Django

### ✅ Changes Made
- [x] Added appropriate `.gitignore` syntax entries for **TypeScript**, **FastAPI**, and **Django**
- [x] Maintained **alphabetical order** in `options.js` to ensure consistency and readability
- [x] Followed all **contribution guidelines** meticulously

---

Thank you for reviewing the changes! Let me know if any refinements are needed. 💡

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added comprehensive .gitignore templates for Django, FastAPI, and TypeScript projects.
  * Expanded the list of available .gitignore templates to include Django, FastAPI, and TypeScript.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

wazeerc commented on src/templates/Django.gitignore:
```diff
@@ -0,0 +1,105 @@
+staticfiles/
+static/
+
+# Django migrations
+*/migrations/*
+!*/migrations/__init__.py
+
```
@prathamhanda consider this

wazeerc commented on package-lock.json:
```diff
@@ -4900,4 +4900,4 @@
 	}
-}
\ No newline at end of file
+}
```
@prathamhanda please undo this change

wazeerc commented on src/lib/options.ts:
```diff
@@ -14,15 +14,16 @@ export default [
 	"ArchLinuxPackages",
-	"C++",
 	"C",
+	"C++",
```
@prathamhanda please don't change the alphabetical order in this file

wazeerc commented on src/lib/options.ts:
```diff
@@ -40,19 +42,20 @@ export default [
 	"Java",
-	"Jekyll",
 	"JENKINS_HOME",
+	"Jekyll",
```
@prathamhanda please don't change the alphabetical order in this file

wazeerc commented on src/lib/options.ts:
```diff
@@ -40,19 +42,20 @@ export default [
 	"Nuxt",
-	"Objective-C",
 	"OCaml",
+	"Objective-C",
```
@prathamhanda please don't change the alphabetical order in this file

wazeerc commented on src/lib/options.ts:
```diff
@@ -69,12 +72,12 @@ export default [
 	"UnrealEngine",
-	"VisualStudio",
 	"Vite",
+	"VisualStudio",
```
@prathamhanda please don't change the alphabetical order in this file

prathamhanda commented:
Hi @wazeerc,

All requested changes have been made:

Added Django, FastAPI, and TypeScript to the options in their correct alphabetical positions, without disturbing the original order.
Addressed all review feedback and ensured the file matches the previous structure.

Please let me know if anything else is needed. Thanks for your review!

prathamhanda commented:
Hi @wazeerc,

All requested changes have been made:

Added Django, FastAPI, and TypeScript to the .gitignore options in their
correct alphabetical positions, without disturbing the original order.
Addressed all review feedback and ensured the file matches the previous
structure.
Please let me know if anything else is needed. Thanks for your review!

On Tue, Jul 8, 2025 at 4:59 AM wazeerc ***@***.***> wrote:

> Assigned #5 <https://github.com/wazeerc/dot-ignorer/pull/5> to
> @prathamhanda <https://github.com/prathamhanda>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wazeerc/dot-ignorer/pull/5#event-18514410281>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUS55ZLHCPZSFC3ZEL7SRL3HL7FDAVCNFSM6AAAAACA575JS2VHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJYGUYTINBRGAZDQMI>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

wazeerc commented:
@coderabbit review the changes again

wazeerc commented:
@copilot create a new issue for me - "fix the MongoDB placement for overall consistency"

wazeerc commented:
@prathamhanda thanks for your PR! Feel free to contribute more to the project 🚀
